keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -697,43 +699,51 @@ generatedFrom:
   apiGroup: dra.example.com
   uid: foobar-uid
 
-vendorParameters:
-  # A vendor can put any kind of object here to pass the configuration
-  # parameters down to the kubelet plugin. In this case, the vendor
-  # driver simply copied the entire CR. It could also be some
-  # separate, smaller configuration type.
-  #
-  # Beware that ResourceClaimParameters have separate RBAC rules than
-  # the vendor CRD, so information included here may get visible
-  # to more users than the original CRD. Both objects are in the same
-  # namespace.
-  kind: CardParameters
-  apiVersion: dra.example.com/v1alpha1
-  metadata:
-    name: my-parameters
-    namespace: user-namespace
-    uid: foobar-uid
-  ...
-  spec:
-    minimumRuntimeVersion: v12.0.0
-    minimumMemory: 32Gi
-    sharing:
-      strategy: TimeSliced
-
-driverRequests:
-- driverName: cards.dra.example.com
-  requests:
-  # Each entry here is a request for one resource.
-  - namedresources:
+requests:
+- vendorParameters:
+    # A vendor can put any kind of object here to pass the configuration
+    # parameters down to the kubelet plugin. In this case, the vendor
+    # driver simply copied the entire CR. It could also be some
+    # separate, smaller configuration type.
+    #
+    # Beware that ResourceClaimParameters have separate RBAC rules than
+    # the vendor CRD, so information included here may get visible
+    # to more users than the original CRD. Both objects are in the same
+    # namespace.
+    kind: CardParameters
+    apiVersion: dra.example.com/v1alpha1
+    metadata:
+      name: my-parameters
+      namespace: user-namespace
+      uid: foobar-uid
+    ...
+    spec:
+      minimumRuntimeVersion: v12.0.0
+      minimumMemory: 32Gi
+      sharing:
+        strategy: TimeSliced
+  # Each entry here is a request for one resource. Entries could have
+  # their own vendor parameters (not shown here).
+  subRequests:
+  - namedResources:
+      driverName: cards.dra.example.com
```

Comment:
This is the key conceptual change: the driver name is no longer required at the level above "named resources". Instead, instances of "the driver I want to use" are selected via the CEL expression.

This `driverName` is just there to simplify the names of attributes in the CEL expression. I'm not sure whether this is important enough to justify the extra complexity.